setup-without-nix.sh

```diff
@@ -69,4 +69,78 @@ EOF
     sleep 5
     /usr/local/bin/tailscale up --auth-key file:/Volumes/CONFIG/tailscale.token
   fi
+
+  if ! pgrep -qf "buildkite-agent"; then
+    # buildkite-agent
+    curl -sL https://raw.githubusercontent.com/buildkite/agent/main/install.sh -o install-buildkite.sh
+    HOME=/tmp/buildkite-agent-staging bash ./install-buildkite.sh
+
+    mv /tmp/buildkite-agent-staging/.buildkite-agent /var/lib/buildkite-agent
+
+    cat <<EOF > /var/lib/buildkite-agent/buildkite-agent.cfg
+token="$(cat /Volumes/CONFIG/buildkite.token)"
+name="%hostname-%n"
+spawn=4
+meta-data="mac=1,nix=0,system=$arch"
+build-path="/var/lib/buildkite-agent/builds"
+hooks-path="/var/lib/buildkite-agent/hooks"
+EOF
+
+    while [ ! -d /Volumes/CONFIG ]; do
+      echo "Waiting for /Volumes/CONFIG to exist ..."
+      sleep 1
+    done
```

Comment:
Add this wait for tailscale's token too, or just generally before running the script?